packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -448,6 +450,7 @@ function AuthOptions({
             targetId={targetId}
             className={className?.onboardingSignup}
             onboardingSignupButtonSize={onboardingSignupButtonSize}
+            onboardingSignupButtonVariant={onboardingSignupButtonVariant}
```

Comment:
Would these 2 better as a sing json?

Reply:
Hmm, makes sense. I'll change it to accept more button props 👍